prow/config.yaml

```diff
@@ -678,8 +678,8 @@ presubmits:
     context: pull-test-infra-verify-bazel
     branches:
     - master
-    rerun_command: "@k8s-bot pull-test-infra-verify-bazel test this"
-    trigger: "@k8s-bot (pull-test-infra-verify-bazel |verify )?test this"
+    rerun_command: "/test pull-test-infra-verify-bazel"
+    trigger: "(^@k8s-bot (pull-test-infra-verify-bazel |verify )?test this\r?$|^/test( pull-test-infra-verify-bazel| verify)?\r?$)"
```

Comment:
ya! i will remove `/test verify`, `/test pull-test-infra-verify-bazel`, and `@k8s-bot pull-test-infra-verify-bazel test this`.